v2.1/online-schema-changes.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Online Schema Changes
+summary: Update table schemas without external tools or downtime.
+toc: true
+---
+
+CockroachDB's online schema changes provide a simple way to update a table schema without imposing any negative consequences on an application — including downtime. The schema change engine is a built-in feature requiring no additional tools, resources, or ad hoc sequencing of operations.
+
+Benefits of online schema changes include:
+
+- Changes to your table schema happen while the database is running.
+- The schema change runs as a [background job][show-jobs] without holding locks on the underlying table data.
+- Your application's queries can run normally, with no effect on read/write latency. The schema is cached for performance.
+- Your data is kept in a safe, [consistent][consistent] state throughout the entire schema change process.
+
+{{site.data.alerts.callout_success}}
+Support for schema changes within transactions is limited. We recommend doing schema changes outside transactions where possible.  When a schema management tool uses transactions on your behalf, we recommend only doing one schema change operation per transaction.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_info}}
+Schema changes keep your data consistent at all times, but they do not run inside user transactions in the general case. This is necessary so the cluster can remain online and continue to service application reads and writes. For more information, see [Limitations](#limitations).
+{{site.data.alerts.end}}
+
+<span class="version-tag">New in v2.1:</span> You can run schema changes inside the same transaction as a [`CREATE TABLE`][create-table] statement. For more information, [see this example](#run-schema-changes-inside-a-transaction-with-create-table).
```

Comment:
This note feels a bit out of place to me here. I think it might be fine to remove this since you have an explicit example on the page and mention it at the bottom of the limitations section.

Reply:
Fixed (by removing).